crate/server/src/database/tests/database_tests.rs

```diff
@@ -26,7 +27,7 @@ use crate::{
 pub(crate) async fn tx_and_list<DB: Database>(
     db_and_params: &(DB, Option<ExtraDatabaseParams>),
 ) -> KResult<()> {
-    cosmian_logger::log_utils::log_init(None);
+    log_init(None);
```

Comment:
What is this for ?

Reply:
it is supposed to clarify the reading ^^